app/models/user.rb

```diff
@@ -1,4 +1,25 @@
 class User < ActiveRecord::Base
   has_many :bids
-  validates :email, email: true, allow_blank: true
+
+  validates :email, allow_blank: true, email: true
+  validate :validate_credit_card_form
+
+  private
+
+  def validate_credit_card_form
+    return if credit_card_form_url.blank?
```

Comment:
what about checking `if credit_card_form_url.present?` instead ? I think that would be more easy to understand